src/routes/profiles.py

```
@@ -37,7 +37,7 @@ async def create_profile(
     jwt_manager: JWTAuthManagerInterface = Depends(get_jwt_auth_manager),
     db: AsyncSession = Depends(get_db),
     s3_client: S3StorageInterface = Depends(get_s3_storage_client),
-    profile_data: ProfileCreateSchema = Depends(ProfileCreateSchema.from_form),
+    profile_data: ProfileCreateSchema = Form(...),
```

Ruff (0.11.9)

Do not perform function call `Form` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)